history.txt

```diff
@@ -1,6 +1,45 @@
 OMERO version history
 =====================
 
+5.2.2 (February 2016)
+---------------------
+
+A bug-fix release which also introduces some new client features. Improvements
+include:
+
+-  display of ROI masks in OMERO.web image viewer
+-  display of OMERO.tables data for Wells in the OMERO.web right hand panel
+-  'Populate Metadata' script to enable generation of OMERO.tables for
+   Wells is now usable from both OMERO.web and OMERO.insight (note this is
+   still in development and has some limitations)
+-  measurement tool fixes
+-  fixed pixel size metadata and scalebar in OMERO.web image viewer for images
+   with pixel size units other than micrometer
+-  fixed OMERO.web handling of turning off interpolation of pixels
+-  previous and next buttons fixed in OMERO.web image viewer
+-  delete and change group performance improvements
+-  better handling of dates in search
+-  client support for map annotations in OME.tiff
+-  disabled orphaned container feature
+-  OMERO.web clean-up to remove obsolete volume viewer
+
+Developer updates include:
+
+-  Python API examples for creating Polygon and Mask shapes
+-  OMERO.tables documentation extended
+-  updated 'What's New for developers' to clarify that ``pojos`` has been
+   renamed as ``omero.gateway.model``
+-  dynamic scripts functionality documented
```

Comment:
sorry forgot about that: dynamically loading omero.client server settings into HTTP session